iset.mm

```diff
@@ -33291,6 +33296,33 @@ same disjoint variable group (meaning ` A ` cannot depend on ` x ` ) and
       ( cpw wss cuni cv wral sspwuni unissb bitri ) BCDEBFCEAGCEABHBCIABCJK $.
   $}
 
+  $( Characterization of the elements of a double power class: they are exactly
+     the sets whose union is included in that class.  (Contributed by BJ,
+     29-Apr-2021.) $)
+  elpwpw $p |- ( A e. ~P ~P B <-> ( A e. _V /\ U. A C_ B ) ) $=
+    ( cpw wcel cvv wss wa cuni elpwb sspwuni anbi2i bitri ) ABCZCDAEDZAMFZGNAHB
+    FZGAMIOPNABJKL $.
```

Comment:
We should add  ` |- ( A e. ~P ~P B <-> U. A e. ~P B ) ` (from sspwuni and uniexb), and see if it's shorter to prove it from elpwpw or to prove elpwpw from it (in both set/iset).
You do it @jkingdon ? or @avekens ? Or else I do it in a few days ?

Reply:
I don't know, I'm not sure I'm really sold on the value of any of these theorems but I copied them over because they were in set.mm.

Reply:
I think that most of them would need a re-think if we wanted to do this in CZF (although I guess @benjub might know more about CZF than I do, I'm mostly just seeing a power set).

Reply:
Regarding CZF: these results probably remain true in CZF since they do not use that the powerclass of a set is a set.

Reply:
Added the above proposed theorem (in both set/iset) in #3002.